src/sections/dataset/dataset-files/files-table/row-selection/RowSelectionMessage.tsx

```diff
@@ -0,0 +1,51 @@
+import { createRowSelection, RowSelection } from '../useFilesTable'
+import { Button } from '@iqss/dataverse-design-system'
+import { useTranslation } from 'react-i18next'
+import styles from './RowSelectionMessage.module.scss'
+
+interface RowSelectionMessageProps {
+  selectedFilesCount: number
+  totalFilesCount: number
+  setRowSelection: (rowSelection: RowSelection) => void
+}
+
+const MINIMUM_SELECTED_FILES_TO_SHOW_MESSAGE = 0
+const MINIMUM_FILES_TO_SHOW_MESSAGE = 10
```

Comment:
Just a note, no changes required. In the future it may be interesting to convert these constants to config properties of the application.

Reply:
If we want to give that level of customisation to the files table, yes, we could make all of these part of a config file 👍